components/telemetry-operator/internal/configbuilder/section_builder.go

```diff
@@ -0,0 +1,75 @@
+package configbuilder
```

Comment:
I guess, everything here can be written lower-case, the `sectionBuilder` should not stick to the outside